client/blocks/login/social.jsx

```diff
@@ -32,19 +33,35 @@ class SocialLoginForm extends Component {
 		translate: PropTypes.func.isRequired,
 		loginSocialUser: PropTypes.func.isRequired,
 		linkingSocialService: PropTypes.string,
+		socialService: PropTypes.string,
+		socialServiceResponse: PropTypes.object,
 	};
 
 	static defaultProps = {
 		linkingSocialService: '',
 	};
 
-	handleGoogleResponse = ( response ) => {
-		const { onSuccess, redirectTo } = this.props;
+	handleGoogleResponse = ( response, triggeredByUser = true ) => {
```

Comment:
Shouldn't we update the [`SocialSignupForm` component](https://github.com/Automattic/wp-calypso/blob/38041beb6e5d051d54b578cb318fc896b67e1151/client/components/signup-form/social.jsx#L22-L28) as well?

Reply:
Thanks, forgot about that!